politeiawww/cmswww.go

```diff
@@ -1016,6 +1016,47 @@ func (p *politeiawww) handleProposalBillingDetails(w http.ResponseWriter, r *htt
 	util.RespondWithJSON(w, http.StatusOK, svr)
 }
 
+// makePropsoalsRequestCached passes a request to makeProposalsRequest
```

Comment:
Still a little off.  Function delcaration coments should start with `// functionName...`, use full sentences, and use the full 80 columns.  When in doubt, match the existing formatting of the repo you're contributuing to.  All the other types of comment formatting is up to you.

Reply:
Ok now ?

Reply:
Yup, looks good.